src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/tests/Search.test.js

```diff
@@ -11,15 +11,35 @@ jest.mock('../../../utils/Translator', () => ({
 
 test('The component should render collapsed', () => {
     const search = render(
-        <Search onSearch={jest.fn()} />
+        <Search onSearch={jest.fn()} value={null} />
     );
 
     expect(search).toMatchSnapshot();
 });
 
+test('The component should render not collapsed when value is given', () => {
+    const search = shallow(
+        <Search onSearch={jest.fn()} value={'search-string'} />
```

Comment:
Until now we have written `value="search-string"`. Also valid for the other tests here.